framework-plugins/lisk-framework-chain-connector-plugin/src/types.ts

```diff
@@ -43,11 +43,16 @@ export interface ValidatorsData {
 	validatorsHash: Buffer;
 }
 
+interface CcmWithHeight {
+	ccm: CCMsg;
+	height: number;
+}
+
```

Comment:
maybe we can remove it as we have a db store for ccms to be retrieved or set by height